tests/Unit/LockboxServiceTest.php

```
@@ -0,0 +1,51 @@
+        $user->forceFill([
+            'name' => 'Test',
+            'email' => 'test@example.com',
+            'password' => bcrypt('password'),
+        ])->save();
+
+        $key = random_bytes(32);
+        $encrypter = new Encrypter($key, config('app.cipher'));
+
+        $manager = new class ($encrypter) extends LockboxManager {
+            public function __construct(private Encrypter $encrypter) {}
+            public function forUser(User $user, ?string $input = null, ?string $providerClass = null): Encrypter
```

PHPMD (2.15.0)

Avoid unused parameters such as '$input'. (undefined)

(UnusedFormalParameter)